### PR TITLE
Add explicit error when too many login attempts

### PR DESCRIPTION
**Pull request by suricactus**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2957 Add explicit error when too many login attempts](https://app.clickup.com/t/2192114/QF-2957)

File: docker-app/qfieldcloud/core/exceptions.py [hunk 1]
```diff
@@ -58,6 +58,14 @@ class NotAuthenticatedError(QFieldCloudException):
     status_code = status.HTTP_401_UNAUTHORIZED
 
 
+class TooManyLoginAttemptsError(QFieldCloudException):
+    """Raised when QFieldCloud had too many failed login attempts."""
+
+    code = "too_many_failed_login_attempts"
+    message = "Too many failed login attempts!"
```
Comment (m-kuhn) [inline docker-app/qfieldcloud/core/exceptions.py:65]: Do we handle translation?

Comment (suricactus) [reply docker-app/qfieldcloud/core/exceptions.py:65]: We never did so far. We can do it in a follow up PR.

**Review by m-kuhn — APPROVED** [review]
Please check the code formatting

**Comment by suricactus** [comment]
This is fixed in master, I branched before being fixed.

**Pull request merged**